### PR TITLE
add report description to UCR report headers

### PR DESCRIPTION
**Pull request by czue**
Looks like this. Open to UI suggestions from the gurus. @biyeun @orangejenny 

image

### CODE REVIEW

**Comment by czue** [comment]
cc @calellowitz

File: corehq/apps/reports_core/templates/reports_core/base_template_new.html [hunk 1]
```diff
@@ -139,6 +138,7 @@
 {% endblock report_alerts %}
 {% block main_column_content %}
 <div class="report-controls">
+    {% if report.spec.description %}<h1><small>{{ report.spec.description }}</small></h1>{% endif %}
```
Comment (benrudolph) [inline corehq/apps/reports_core/templates/reports_core/base_template_new.html]: `small` in an `h1` seems odd to me. seems like you could use `h2/3` and the `lead` class

**Comment by czue** [comment]
@benrudolph updated to use the `lead` class (and converted to `p` which seemed a little more semantically accurate). now it looks like this (kinda not distinct from the "report filters" heading - which - as a separate point, could possibly just be removed altogether). i'm out of my element on this stuff.

image

**Comment by benrudolph** [comment]
hmm yeah sorry for the mislead. i'm OK with either since this is probably going to be replaced by the b3 upgrade and it's more important to show the description. feel free to self-merge

**Pull request merged**